ibstore/_tests/unit_tests/test_store.py

```diff
@@ -19,6 +19,8 @@ def test_from_qcsubmit(small_collection):
     # Ensure a new object can be created from the same database
     assert len(MoleculeStore("foo.sqlite")) == len(store)
 
+    os.remove("foo.sqlite")
+
```

Comment:
What do you think about wrapping the contents of this test into a `temporary_cd` context manager? Throwing around `foo.sqlite` on disk is just me being sloppy.

https://github.com/openforcefield/openff-utilities/blob/v0.1.9/openff/utilities/utilities.py#L155

Reply:
That's a great idea. I tried using a NamedTemporaryFile like the tests below it, but that creates the file and triggers the DatabaseExistsError. `temporary_cd` seems perfect.